### DISCUSSION TITLE
Dealing with age-restricted videos

### DISCUSSION THREAD

**Discussion by k0zyrev** [FreeTubeApp/FreeTube#6218] [General]
Youtube completely blocked any workarounds that allowed accessing age-restricted videos and demands proper login. I'm asking if there are any remote future plans to implement something like an import of a token or cookie from a logged-in account in a browser and using that login data to access restricted videos?

I'm fairly certain something like that is implemented on at least some invidious instances where they generate individual tokens for users which allows them not to get ip-banned. Yes, it's a bit different in purpose, but the idea is roughly the same.

**Comment by absidue** [comment]
The Invidious thing is very different, it has nothing to do with being logged in, those are just the anonymous tracking/session identifiers that YouTube's own website also creates when you are logged out, FreeTube generates a different one for each video, so it is harder to track you.

I would also like to direct you to all of our existing comments about logging in with a Google account and why we will not be adding it.

**Comment by Majstor** [comment]
Some apps that use Google Service like Aurora Store (Android) deal with an "anonymous" account. Isn't it a possible way ?
